.github/workflows/gobra.yml

```diff
@@ -265,7 +265,7 @@ jobs:
         uses: viperproject/gobra-action@main
         with:
           packages: 'pkg/slayers/path/scion'
-          timeout: 5m
+          timeout: 15m
```

Comment:
why are we increasing the timeout here? So far, we do not really need it here right?

Reply:
For some reason it timed out once but now it seems stable now. So I will revert that change.